packages/viem/typedoc.json

```diff
@@ -1,16 +1,17 @@
 {
-    "$schema": "https://typedoc.org/schema.json",
-    "out": "./docs",
-    "plugin": ["typedoc-plugin-markdown"],
-    "entryPoints": ["./src/index.ts", "./src/actions/interop/index.ts", "./src/chains/index.ts"],
-    "exclude": [
-        "**/*.spec.ts",
-        "./src/decorators/**",
-        "./src/test/**"
-    ],
-    "readme": "none",
-    "excludeInternal": true,
-    "textContentMappings": {
-        "title.memberPage": "{name}",
-    }
-}
\ No newline at end of file
+  "$schema": "https://typedoc.org/schema.json",
+  "out": "./docs",
+  "plugin": ["typedoc-plugin-markdown"],
+  "entryPoints": [
+    "./src/index.ts",
+    "./src/actions/index.ts",
```

Comment:
This line is the only real addition. Everything else in this file is prettier, everything else in this PR is the newly generated docs